testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/mgmt/PooledConnectionFactoryStatisticsTestCase.java

```diff
@@ -86,10 +88,6 @@ public void testStatistics() throws Exception {
 
             bean.closeConnection();
             assertEquals(0, readStatistic("InUseCount"));
-
-        } finally {
-            disableStatistics();
-            checkStatisticsAreDisabled();
```

Comment:
The call to `checkStatisticsAreDisabled();` should still be present after the try-with-resources block

Reply:
Why? We know they are disabled by default, so when we restore they will go back to the default state.